recipes/jupyter-syncthing-proxy/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "jupyter-syncthing-proxy" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyter-syncthing-proxy-{{ version }}.tar.gz
+  sha256: df3d74d140e97ed73a5e4c3862564799dd92c53e8cfd83c8ef29a5d95c82dc27
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - jupyter-server-proxy
+    - python >=3.8
+
+test:
+  imports:
+    - jupyter_syncthing_proxy
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/yuvipanda/jupyter-syncthing-proxy
+  summary: Run syncthing inside JupyterHub
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - yuvipanda
```

Comment:
@yuvipanda, ok if I list you as a conda-forge recipe maintainer for `jupyter-syncthing-proxy` here (need an explicit yes/no answer)?

Reply:
@weiji14 ok with me!